src/_lib/sass/src/_buttons.scss

```diff
@@ -85,58 +77,50 @@ a.btn {
 }
 .btn-danger {
     @include button-variant(
-        #ffffff,
-        #ffffff,
+        $text-color,
+        $text-color,
         get-color('tertiary'),
-        get-color('tertiary', dark));
+        get-color('tertiary', light));
 }
 .btn-alt {
     @include button-variant(
-        get-color('gray', 4x-light),
+        $text-color,
         get-color('primary', 2x-light),
         transparent,
         transparent,
-        get-color('gray', 4x-light),
+        $text-color,
         get-color('primary', 2x-light));
 }
 .btn-ghost {
     @include button-variant(
-        get-color('gray', 4x-light),
-        get-color('primary', 2x-light),
+        $text-color,
+        $text-color,
         transparent,
-        get-color("gray", dark));
+        rgba($text-color, 0.1));
 }
 .btn-dropdown {
-    @include button-variant(
-        #ffffff,
-        #ffffff,
-        get-color("gray", base),
-        get-color("gray", dark));
+    @extend .btn-secondary;
 }
 .btn-dropdown-inverse {
-    @include button-variant(
-        get-color("gray", base),
-        get-color("gray", base),
-        #ffffff,
-        #ffffff);
+    @extend .btn-secondary-inverse;
 }
```

Comment:
By the looks of this it appears that btn-dropdown and btn-dropdown-inverse should be removed as classes and their occurences in the HTML replaced with btn-secondary and btn-secondary-inverse. If you think this is too much as part of the el layers story it would be good to add it to the tech debt backlog

Reply:
I have a separate story for dropdowns under EL, can do it there. Will let you know if I decide to move it to the FE tech debt once I get to it.